proposals/rich-content-graphql-api.md

```diff
@@ -1,380 +0,0 @@
-# Common GraphQL Representation of Rich Content
```

Comment:
This proposal need to be moved to design-documents folder, not removed.

Reply:
Fixed